django/db/backends/oracle/creation.py

```diff
@@ -216,7 +227,15 @@ def _create_test_user(self, cursor, parameters, verbosity):
                      CREATE TRIGGER
                TO %(user)s""",
         ]
-        self._execute_statements(cursor, statements, parameters, verbosity)
+        try:
+            # Statement can fail when keepdb is on
+            self._execute_statements(cursor, statements, parameters, verbosity, allow_quiet_fail=keepdb)
+        except DatabaseError as err:
+            description = str(err)
+            # Avoid user already exists error when keepdb is on
+            if 'ORA-01920' not in description or not keepdb:
+                raise
+
```

Comment:
Note that with the two instances you added, there are now 3 instances in the file of almost the same pattern: 
```python
        try:
            # Statement can fail when keepdb is on
            self._execute_statements(cursor, statements, parameters, verbosity, allow_quiet_fail=keepdb)
        except DatabaseError as err:
            description = str(err)
            # Avoid user already exists error when keepdb is on
            if specific_error not in description or not keepdb:
                raise
```
I think the 3 instances can be folded into `_execute_statements()` (and your new instances can also print a message when execution fails acceptably, when `verbosity>=2`, like the existing one does).